test/demo/eventListeners.inttest.ts

```diff
@@ -88,8 +88,9 @@ describe('Click listeners', () => {
                 await noteCards[0].click();
                 await delay(Duration.EVENT_PROPAGATED);
 
-                // Listener was triggered once with the correct data
-                expect(actionListener).toHaveBeenCalledTimes(1);
+                /* Listener was triggered twice with the correct data
+                    (once for notification-action and once for notification-action triggered by notification-closed events)*/
+                expect(actionListener).toHaveBeenCalledTimes(2);
```

Comment:
Let's stick with either

    /**
     * This style
     */

or

    // This style

Think the latter in this case, and if it needs to be two lines that's fine

Reply:
Get rid of the stray space, then this is good